android-components/components/compose/awesomebar/build.gradle

```diff
@@ -49,7 +49,8 @@ dependencies {
 
     implementation ComponentsDependencies.androidx_core_ktx
     implementation ComponentsDependencies.androidx_compose_ui
-    implementation ComponentsDependencies.androidx_compose_ui_tooling
+    debugImplementation ComponentsDependencies.androidx_compose_ui_tooling
```

Comment:
nit: we have separate groupings to make it easier to grok. You can either:
* Move the `debugImplementation` to a new group to signify "debug" dependencies.
* Move the `androidx_compose_ui_tooling` and `androidx_compose_ui_tooling_preview` dependencies to a new group to signify "compose_ui_tooling" dependencies.

Reply:
Spoke offline: these will be updated as part of a future change.